pkg/trace/api/api.go

```diff
@@ -71,13 +72,13 @@ type HTTPReceiver struct {
 	Stats       *info.ReceiverStats
 	RateLimiter *rateLimiter
 
-	out              chan *Payload
-	conf             *config.AgentConfig
-	dynConf          *sampler.DynamicConfig
-	server           *http.Server
-	statsProcessor   StatsProcessor
-	appsecHandler    http.Handler
-	configSubscriber *config.Subscriber
+	out            chan *Payload
+	conf           *config.AgentConfig
+	dynConf        *sampler.DynamicConfig
+	server         *http.Server
+	statsProcessor StatsProcessor
+	appsecHandler  http.Handler
+	grpc           pbgo.AgentSecureClient
```

Comment:
```suggestion
	coreClient           pbgo.AgentSecureClient // gRPC client to core agent process
```
Is this correct? If yes, please rename it.

Reply:
done